File: libs/power-log-parser/src/lib/parsers/corpses-spent-this-game-parser.ts
import { GameTag } from '@firestone-hs/reference-data';
import { AbstractBasicTagChangeParser } from './abstract-basic-tag-change-parser';
import { ParserState } from '../state/parser-state';
import type { StateFacade } from '../state/state-facade';

export class CorpsesSpentThisGameParser extends AbstractBasicTagChangeParser {
	readonly ParserName = 'CorpsesSpentThisGameParser';

	constructor(parserState: ParserState, facade: StateFacade) {
		super(parserState, facade, GameTag.CORPSES_SPENT_THIS_GAME, 'CORPSES_SPENT_THIS_GAME_CHANGED');
	}
}

File: tools/linux-helpers/hs-converter/HearthstoneReplays/Events/Parsers/CorpsesSpentThisGameParser.cs
using HearthstoneReplays.Parser;
using HearthstoneReplays.Enums;

namespace HearthstoneReplays.Events.Parsers
{
    public class CorposesSpentThisGameParser : AbstractBasicTagChangeParser
    {
        public CorposesSpentThisGameParser(ParserState ParserState, StateFacade facade) 
            : base(ParserState, facade, GameTag.CORPSES_SPENT_THIS_GAME, "CORPSES_SPENT_THIS_GAME_CHANGED")
        {
        }
    }
}
